src/components/Modals/EditExpertProfileModal.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useContext, useRef, useState } from 'react';
+import DialogModalBase from './DialogModalBase';
+import styles from './EditExpertProfileModal.module.scss';
+import Context from '../../context';
+import { ModalContext } from '../../context/ModalContext';
+
+const accentColor = '#4E6AE6';
+
+const EditExpertProfileModal = (props) => {
+  const [email, setEmail] = useState(null);
+  const [expertise, setExpertise] = useState(null);
+  const [description, setDescription] = useState(null);
+  const [expertiseTags, setExpertiseTags] = useState(null);
+  const [visibility, setVisibility] = useState(null);
+
+  const [fieldsMissing, setFieldsMissing] = useState(false);
+
+  const apiContext = useContext(Context.Api);
+  const { user } = useContext(Context.User);
+  const modalContext = useContext(ModalContext);
+
+  const [Avatar, setAvatar] = useState(props.avatar);
+
+  const avatarUploader = useRef(null);
+
+  function closeModal() {
+    modalContext.setOpenedModal(null);
+    // can be optional
+    [
+      setEmail,
+      setExpertise,
+      setDescription,
+      setExpertiseTags,
+      setVisibility,
+    ].map((method) => typeof method === 'function' && method(null));
+  }
+
+  const putExpertProfile = () => {
+    if (
+      email == null ||
+      expertise == null ||
+      description == null ||
+      expertiseTags == null ||
+      visibility == null
+    ) {
+      setFieldsMissing(true);
+      return;
+    }
+    apiContext
+      .updateJufoExpert(user.id, {
+        contactEmail: email,
+        description,
+        expertiseTags: expertiseTags.split(','),
+        active: visibility,
+      })
+      .then(() => closeModal());
+  };
+
+  return (
+    <DialogModalBase accentColor={accentColor}>
+      <DialogModalBase.Modal modalName="editExpertProfileModal">
+        <DialogModalBase.Header>
+          <div className={styles.avatarWrapper}>
+            <Avatar className={styles.avatar} />
```

Comment:
As long as we don't have any avatar upload we omit this.

Reply:
Fixed in 3a687e1458e47e8e60698eadfb36d91abd70d3f1